spread/runner.go

```diff
@@ -631,7 +631,7 @@ func (r *Runner) worker(backend *Backend, system *System, order []int) {
 				debug = ""
 				repeat = -1
 			}
-			if !abend && !r.options.Restore && repeat == 0 {
+			if !abend && !r.options.Restore && repeat <= 0 {
```

Comment:
It's a very good practice to review one's own suggested change before others do.

Reply:
This change is to make residue work when the test fails, currently it was just leaving the residue when the test passed. I noticed that when I ran the residue test and it was failing.

Reply:
This seems unrelated to this PR purpose.

Reply:
@niemeyer, Without this change the residue test is going to fail on google and qemu as well. I dont see any problem to revert that and add it in another PR. What do you prefer?

Reply:
Ack, thanks for the explanation online.